src/_data/platforms.yml

```diff
@@ -93,10 +93,10 @@
       support_level: production
       type: framework
       name: Android
-      doc_link: /clients/java/modules/android/
+      doc_link: /platforms/android/
       wizard_parent: java
```

Comment:
`wizard_parent: java` I don't know what it is used for, but moving away from the java client to its own platform, this line seems wrong

Reply:
I think this is not really relevant, only for the code parts that we have spread everywhere.